plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -201,6 +201,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
 
       std::atomic<int32_t>                                      _max_transaction_time_ms; // modified by app thread, read by net_plugin thread pool
       fc::microseconds                                          _max_irreversible_block_age_us;
+      static uint64_t                                           _max_incoming_transaction_queue_size;
```

Comment:
it seems odd that this is `static`.  it is only used (other than configuration) inside the non-static member of type `incoming_transaction_queue`.  Can it just be stored there?  meaning configuration pushes it into that class.

Reply:
I made it static so the embedded class has direct access to it. I'll plumb it in.